python/pyspark/sql/dataframe.py

```diff
@@ -78,6 +78,12 @@ def __init__(self, jdf, sql_ctx):
         self.is_cached = False
         self._schema = None  # initialized lazily
         self._lazy_rdd = None
+        self._eager_eval = sql_ctx.getConf(
+            "spark.jupyter.eagerEval.enabled", "false").lower() == "true"
```

Comment:
let's add all these to documentation too

Reply:
Got it. Do it in next commit. Thanks for reminding.